micro_ros_setup/config/zephyr/generic/create.sh

```diff
@@ -0,0 +1,52 @@
+# To be removed later, please use package.xml
```

Comment:
Seconded ;-)

Reply:
Check 

https://github.com/micro-ROS/micro-ros-build/blob/d56aa8493a7887855e479e3adf19a4bb331c004a/micro_ros_setup/config/zephyr/generic/package.xml

and 

https://github.com/micro-ROS/micro-ros-build/blob/d56aa8493a7887855e479e3adf19a4bb331c004a/micro_ros_setup/config/zephyr/generic/create.sh